index.bs

```diff
@@ -1538,6 +1546,10 @@ the following table:
   <tr>
    <td><dfn>\[[stream]]</dfn>
    <td class="non-normative">The {{ReadableStream}} instance controlled
+  <tr>
+   <td><dfn>\[[isOwning]]</dfn>
+   <td class="non-normative">A boolean flag indicating whether to take ownership of enqueued chunks
+   via transfer or serialization.
```

Comment:
Nit-pick: this table is sorted alphabetically, so this should go between `[[closeRequested]]` and `[[pullAgain]]`.

Reply:
Done